### PR TITLE
Add initContainer securityContext to helm chart

### PR DESCRIPTION
**Pull request by funkypenguin**
# Description

Hey guys! This PR simply uses the existing logic in the helm chart to insert a securityContext for each of the initContainers. We stumbled across this when trying to update to the latest chart version - our CI uses kube-score to lint our manifests, and wouldn't permit creating a pod without securityContext for its initContainers :)

## Type of change

*Please select one of the options below.*

- New feature (non-breaking change which adds functionality)

# Discussion

*Are there any design details that you would like to discuss further?*

# Testing

Successfully validate charts against kube-score, and deployed into multiple clusters without incident.

*Do we need to perform additional testing once this is merged, or perform in a larger testing environment?*

No

# Documentation

*Did you update relevant documentation within this repository?*

From what I can tell, the helm chart is not documented to the degree of detail which would require any changes resulting from this PR. If I'm wrong, I'm happy to update the docs if you point me to them.

# Follow-up

*Are there any follow-up issues that we should pursue in the future?* No

*Does this introduce new defaults that we should re-evaluate in the future?* No

### CODE REVIEW

File: helm/fdb-operator/values.yaml [hunk 1]
```diff
@@ -74,3 +74,11 @@ initContainersResources:
...
     cpu: 10m
     memory: 50Mi
+
+initContainerSecurityContext:
+  allowPrivilegeEscalation: false
+  privileged: false
+  capabilities:
+    drop:
+    - all
+  readOnlyRootFilesystem: true
```
Comment (johscheuer) [inline helm/fdb-operator/values.yaml:84]: Missing newline

**Review by johscheuer — CHANGES_REQUESTED** [review]
Only one small thing otherwise LGTM 👍

**Review by johscheuer — COMMENTED** [review]
Friendly ping :)

**Comment by funkypenguin** [comment]
Oops! Friendly pong :)

**Review by johscheuer — APPROVED** [review]
LGTM 👍

**Pull request merged**